### ISSUE TITLE
[BUG] Detecting Systemd on Linux is ropey

### ISSUE THREAD

**Issue by aries1980** [SamSchott/maestral#199]
**Describe the bug**

On Linux, when Maestral starts, it assumes the `ps -p 1` is a valid command, but it depends on the implementation. E.g.
on Alpine Linux, which runs the "Busybox" shell (aka `ash`), the `-p` option does not exist.

The issue is here:
https://github.com/SamSchott/maestral/blame/5092f986eb6f6c3e13f9e4263c2b71a636513bf0/maestral/utils/autostart.py#L391

**To Reproduce**

Run the v1.2.1 Docker image:
```
docker run -it maestraldbx/maestral:1.2.1
```
then look for the error:
```
subprocess.CalledProcessError: Command '['ps', '-p', '1']' returned non-zero exit status 1.
```

**Expected behaviour**
Start automatically without an error.

**System (please complete the following information):**
 - Maestral version: 1.2.1
 - OS: Alpine Linux (the Docker image)

**Additional context**

I think this should be in a try-except block to handle errors.

**Comment by SamSchott** [comment]
This seems to occur when trying to update existing autostart entries after an upgrade to v1.2.1:

https://github.com/SamSchott/maestral/blob/08550104711ff06e5906c85287954f048abf2711/maestral/main.py#L1337-L1347

The autostart backend detection has not changed between v1.2.0 and v1.2.1.

**Comment by SamSchott** [comment]
Fixed in v1.2.2.
